src/renderer/cell.zig

```diff
@@ -246,8 +247,10 @@ pub fn fgMode(
             const cell = cell_pin.rowAndCell().cell;
             const cp = cell.codepoint();
 
-            if (!ziglyph.general_category.isPrivateUse(cp) and
-                !ziglyph.blocks.isDingbats(cp))
+            // If it's not Private Use (Co) or Dingbats (0x2700-0x27bf), use
+            // normal mode.
+            if (GeneralCategories.gc(zg.general_categories, cp) != .Co and
+                !(cp >= 0x2700 and cp <= 0x27bf))
```

Comment:
`zg` doesn't have a module replacing `ziglyph.blocks`, but since we're only using it here, I didn't PR adding it